codespell_lib/data/dictionary.txt

```diff
@@ -19452,78 +19460,161 @@ gayity->gaiety
 gaysha->geisha
 gayshas->geishas
 geeneric->generic
+geenral->general
+geenrally->generally
 geenrate->generate
 geenrated->generated
 geenrates->generates
+geenrating->generating
 geenration->generation
 geenrational->generational
+geenrations->generations
+geenrator->generator
+geenrators->generators
+geenric->generic
+geenrics->generics
 geeoteen->guillotine
 geeral->general
 gemetrical->geometrical
 gemetry->geometry
 gemoetry->geometry
 gemometric->geometric
+genaral->general
+genarally->generally
 genarate->generate
 genarated->generated
 genarates->generates
 genarating->generating
 genaration->generation
+genarations->generations
+genarator->generator
+genarators->generators
+genaric->generic
+genarics->generics
 genearal->general
 genearally->generally
+genearate->generate
+genearated->generated
+genearates->generates
+genearating->generating
+genearation->generation
+genearations->generations
+genearator->generator
+genearators->generators
+genearic->generic
+genearics->generics
+genearl->general
+genearlly->generally
+genearte->generate
 genearted->generated
+geneartes->generates
+genearting->generating
+geneartion->generation
+geneartions->generations
+geneartor->generator
+geneartors->generators
 geneate->generate
 geneated->generated
 geneates->generates
 geneating->generating
 geneation->generation
+geneations->generations
+geneator->generator
+geneators->generators
 geneic->generic, genetic,
 geneological->genealogical
 geneologies->genealogies
 geneology->genealogy
 generaates->generates
+generae->generate
```

Comment:
Perhaps `genera`, the plural of `genus`?
```
generae->generate, genera,
```
But then it's probably uncommon. Thoughts?

Reply:
I couldn't find an example of `generae->genera`, so I won't include this.